source/ui/datasources/ui_blockedplayers_datasource.cpp

```diff
@@ -0,0 +1,44 @@
+#include "ui_precompiled.h"
+#include "kernel/ui_common.h"
+#include "datasources/ui_blockedplayers_datasource.h"
+
+#define TABLE_NAME "blockedplayers"
+
+namespace WSWUI
+{
+    BlockedPlayersDataSource::BlockedPlayersDataSource():Rocket::Controls::DataSource( "blockedplayers" )
+	{
+		const char *blocked = trap::Cvar_String("cg_chatBlocklist");
+		// stored in the format `:name      steamid` (name is padded out to match max)
```

Comment:
but why.

Reply:
i'm rewriting this part later,  was a temporary solution